### PR TITLE
feat: output registry URI if single registry ID is provided as input

### PR DESCRIPTION
**Pull request by clareliguori**
Related to #48, 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: index.test.js [hunk 1]
```diff
@@ -85,6 +85,37 @@ describe('Login to ECR', () => {
             expect.anything());
     });
 
+    test('outputs the registry ID if a single registry is provided in the input', async () => {
+        core.getInput = jest.fn().mockReturnValueOnce('111111111111');
```
Comment (allisaurus) [inline index.test.js:89]: should the mocked return value be `registries: { with: [1111111111] }` ?

Comment (clareliguori) [reply index.test.js:89]: Do you mean to mock the configuration: 
```
      - name: Login to Amazon ECR
        id: login-ecr
        uses: aws-actions/amazon-ecr-login@v1
        with:
          registries: 1111111111
```

The GH actions toolkit takes care of parsing the config, and then the action just does core.getInput('registries')

**Comment by allisaurus** [comment]
I [used this update](https://github.com/allisaurus/ecs-first-run/runs/711505249?check_suite_focus=true) with the `registries` field configured as in #48 and am still seeing a failure on the build/tag/push step.`ECR_REGISTRY` appears blank, despite referencing the output of the `login-ecr` file.

Is it possible that setting `authTokenRequest.registryIds` results in a response that would cause [this check](https://github.com/aws-actions/amazon-ecr-login/pull/50/files#diff-168726dbe96b3ce427e7fedce31bb0bcR32) to not work?

**Comment by clareliguori** [comment]
@allisaurus ah, looks like my branch's changes aren't packaged into dist folder, so your action isn't running the change.

**Comment by allisaurus** [comment]
Ah ha! That did it, it's working no problem now. :shipit:

**Pull request merged**